roles/tftp-server/defaults/main.yml

```diff
@@ -17,6 +15,29 @@ user_name: videoteam
 playbook_repo: https://anonscm.debian.org/cgit/debconf-video/ansible.git
 playbook_branch: master
 
-# To replace the stock inventory with your own, point at your own github repo
+# To replace the stock inventory with your own,
+#   point at your own github repo
 #inventory_repo: https://gitlab.com/yourname/ansible-inventory
 #inventory_branch: master
+
+
+# boot image sources
+debian_host: ftp.debian.org
+ubuntu_host: archive.ubuntu.com
+
+debian_suites: [stretch, sid]
```

Comment:
sid doesn't have installer images, does it?

Reply:
it does:
http://ftp.debian.org/debian/dists/sid/main/installer-amd64/current/images/netboot/debian-installer/amd64/

Reply:
Do they work? I wouldn't expect that the udeb stack is kept particularly installable

Reply:
This my make more sense with ubuntu given there are many stable releases that are reasonable.  like when DVswitch worked on Onric but nothing after that.  ever. 

does sid installer work?  just tested, currently no.   installer kernel version  and modules mismatch.   
but as time goes on it becomes more likely to work.
more debugging steps: something is broken, lets see if it works with the next release (what I have been doing with ubuntu)  If it does, it somewhat confirms what I am doing can work, but likely some dependency isn't right.

Or it all crashes and I report a bug in the installer.    Another smooth Yak.

Reply:
>  but as time goes on it becomes more likely to work.

I don't think debian has ever really tried to have sid be installable.

Reply:
oh, buster is what I wanted.  I got sid/buster flipped.  

http://ftp.debian.org/debian/dists/buster/main/installer-amd64/current/images/netboot/debian-installer/amd64/
Last modified 2017-08-28
tried it, kernel module version mismatch. 
do you expect this to ever work?

in the interest of getting stuff merged, I am OK with the default being stretch.